src/test/java/uk/gov/pay/connector/service/GatewayClientTest.java

```diff
@@ -49,6 +52,9 @@ public class GatewayClientTest {
 
     private MediaType mediaType = MediaType.APPLICATION_XML_TYPE;
 
+    private final CollectorRegistry collectorRegistry = CollectorRegistry.defaultRegistry;
+    private String[] labelNames = new String[] {"gatewayName", "gatewayAccountType", "orderRequestType"};
```

Comment:
(minor) Formatting in this file for additional spaces.
```suggestion
    private String[] labelNames = new String[]{"gatewayName", "gatewayAccountType", "orderRequestType"};
```